test/jdk/javax/swing/JTextArea/4697612/bug4697612.java

```diff
@@ -224,7 +224,7 @@ private static void createAndShowGUI() {
         text = new JTextArea();
         try {
             InputStream is =
-                    bug4697612.class.getResourceAsStream("bug4697612.txt");
+                    bug4697612.class.getResourceAsStream("bug4697612.java");
```

Comment:
the .txt file as well as mentioning applets really wasn't appropriate to be in the Open JDK.
I've replaced it with the src of the test - all it needs is a file to scroll and this is very similar in size.